specifyweb/frontend/js_src/lib/components/Attachments/Dialog.tsx

```diff
@@ -91,12 +95,14 @@ export function AttachmentDialog({
     >
       <div className="flex h-full gap-4">
         {/* FEATURE: keyboard navigation support */}
-        <Button.Icon
-          className="p-4"
-          icon="chevronLeft"
-          title={commonText.previous()}
-          onClick={handlePrevious}
-        />
+        {isFirst ? null : (
```

Comment:
why not just hide the button if `handlePrevious` is undefined?
same for the other button - that will remove the need for the 2 new props

Reply:
but really, I would advise against dynamically hiding or showing buttons in a dialog:
this can cause the dialog width to change, and so for example, if user was going to click the button twice rapidly to go left two times, if the button has disappeared on the 2nd click, the user may end up clicking outside the dialog accidentally - thus causing the dialog to close.
@grantfitzsimmons can tell you more about this UX issue

besides that there is a smaller annoyance but still an issue: buttons appearing and disappearing dynamically on the sides of the dialog, mean the dialog content can change it's position, which causes layout shift needlessly

overall, I don't think any of these issues are worth saving a few pixels. @grantfitzsimmons what do you think?

Reply:
The left and right buttons should be always visible, even when there is not another attachment to the right or left. It should just be disabled. Max is right about the usability issue with this!

Reply:
actually, there might be another issue with buttons disappearing:
if an element is focused, and that element disappears, the focus is lost, and so get's reset to the first focusable element on the page.
in this case, if user clicks on the button using the keyboard, and the next record happens to be last, the button would disappear and focus is lost